src/components/App.tsx

```diff
@@ -29,6 +30,20 @@ interface IAppProps extends RouteComponentProps<any> {
 @inject('mapStore', 'loginStore')
 @observer
 class App extends React.Component<IAppProps, IAppState> {
+    async componentWillMount() {
+        const pathName = navigator.getPathName();
```

Comment:
Extract this to new function, currently it is difficult to fast know what it does, when the function name is just componentWillMount()